<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchjob</class>
 <widget class="QDialog" name="searchjob">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_sjob">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1261</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_sjoblist">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Available Jobs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_scj">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_ssjob">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>109</y>
     <width>1261</width>
     <height>601</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_sjob">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Job</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_location">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>110</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bhaktapur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bharatpur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Birgunj</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Butwal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dhangadhi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dhulikhel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ghorahi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hetauda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Itahari</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Janakpur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kathmandu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lalitpur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nepalgunj</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pokhara</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LOCATION</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SALARY</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>100</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FROM</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>100</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_keyword">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit { border-radius: 10px; }</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_search">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>821</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_lower">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>130</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_upper">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>130</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_display">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>460</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_sapp">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>341</width>
      <height>471</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply For the Job</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_JobType">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_JobName">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_JobAddress">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Qualification">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Salary">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_JobId">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_name">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Employer">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_test">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_apply">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>520</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Job Description</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>460</y>
      <width>721</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
